.gitignore

```diff
@@ -26,24 +26,6 @@
 /vendor/
 /composer.lock
 
-# Testing library
-/test_config.yml
-/source/Services/
-
-# Editions
-/source/Edition/
-
-# IDE
-/.idea/
-/.phpstorm-oxid.meta.php
-/.ide-helper.php
-
-# OS
```

Comment:
Mac os is generating those files, should be ignored as well

Reply:
ok - so these are all things which are generated in some cases
Didn't know if these are already needed or remained from older versions of oxid ^^
so I wrote it here as a pull request as a hint and you have a look at it ;)